governance/votes/Executive vote - July 2, 2021.md

```diff
@@ -0,0 +1,144 @@
+| LINK-A | 4.00% | 1.00% | -3.00% |
+| YFI-A | 4.00% | 1.00% | -3.00% |
+| UNI-A | 2.00% | 1.00% | -1.00% |
+| AAVE-A | 2.00% | 1.00% | -1.00% |
+| RENBTC-A | 4.00% | 2.00% | -2.00% |
+| COMP-A | 3.00% | 1.00% | -2.00% |
+| BAL-A | 2.00% | 1.00% | -1.00% |
+| UNIV2DAIETH-A | 2.50% | 1.50% | -1.00% |
+| UNIV2USDCETH-A | 3.50% | 2.00% | -1.50% |
+| UNIV2DAIUSDC-A | 0.50% | 0.00% | -0.50% |
+| UNIV2WBTCETH-A | 3.50% | 2.00% | -1.50% |
+| UNIV2UNIETH-A | 4.00% | 2.00% | -2.00% |
+| UNIV2ETHUSDT-A | 4.00% | 2.00% | -2.00% |
+
+**Other Changes**
+
+Activate the [Debt Ceiling Instant Access Module](https://community-development.makerdao.com/en/learn/governance/module-dciam) for the PSM-USDC-A vault type with the following parameters:
+- Maximum Debt Ceiling (`line`) = 10 billion DAI.
+- Target Available Debt (`gap`) = 1 billion DAI.
+- Ceiling Increase Cooldown (`ttl`) = 24 hours.
+
+**Voting for this executive proposal will place your MKR in support of the changes and additions outlined above.**
+
+Unless otherwise noted, the changes and additions listed above are subject to the [GSM Pause Delay](https://community-development.makerdao.com/en/learn/governance/param-gsm-pause-delay). This means that if this executive proposal passes, the changes and additions listed above will only become active in the Maker Protocol after the GSM Pause Delay has expired. The GSM Pause Delay is currently set to **48 hours**.
+
+This executive proposal **does not** include an office-hours modifier and can be executed immediately after the GSM Pause Delay has expired.
+
+If this executive proposal does not pass within 30 days, then it will expire and can no longer have any effect on the Maker Protocol.
+
+---
+
+## Proposal Details
+
+### July Core Unit Budget Distributions
+
+Due to the positive outcomes of multiple [Core Unit Budget Proposals](https://mips.makerdao.com/mips/list?&search=$AND(%23core-unit,%20%23budget,%20%23active)&subproposalsMode=true), the following budget allocations will be made to fund operations in the month of July if this executive proposal passes.
+
+**155,000 DAI** will be distributed to the **[Real World Finance (RWA-001)](https://mips.makerdao.com/mips/details/MIP39c2SP1)**  core unit at **0x9e1585d9CA64243CE43D42f7dD7333190F66Ca09**  as specified in their active [budget proposal](https://mips.makerdao.com/mips/details/MIP40c3SP12).
+
+**182,000 DAI** will be distributed to the **[Risk (RISK-001)](https://mips.makerdao.com/mips/details/MIP39c2SP2)**  core unit at **0xd98ef20520048a35EdA9A202137847A62120d2d9**  as specified in their active [budget proposal](https://mips.makerdao.com/mips/details/MIP40c3SP13).
+
+**273,334 DAI** will be distributed to the **[GovAlpha (GOV-001)](https://mips.makerdao.com/mips/details/MIP40c3SP11)** core unit at **0x01D26f8c5cC009868A4BF66E268c17B057fF7A73**  as specified in their active [budget proposal](https://mips.makerdao.com/mips/details/MIP40c3SP3). Note that this includes the GovAlpha continuity budget.
```

Comment:
Links appear to have been mixed up. The first link should be https://mips.makerdao.com/mips/details/MIP39c2SP3

Second link should be https://mips.makerdao.com/mips/details/MIP40c3SP11